apps/earn-protocol/hooks/use-transaction.ts

```
@@ -647,7 +709,38 @@ export const useTransaction = ({
+    tokenBalance,
+    sidebarTransactionType,
+    positionAmount,
+    isDeposit,
+    isWithdraw,
+    isSwitch,
+    selectedSwitchVault,
+  ])
+
+  // refresh the transactions list when the amount changes, while is switching
+  useEffect(() => {
+    if (amount && amount.isGreaterThan(0) && isSwitch) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)